README.md

```diff
@@ -37,15 +37,63 @@ Available frontends to define the folded circuit:
 - [arkworks](https://github.com/arkworks-rs), arkworks contributors
 - [Circom](https://github.com/iden3/circom), iden3, 0Kims Association
 
-
 ## Usage
 
+Detailed usage and design documentation can be found [here](https://privacy-scaling-explorations.github.io/sonobe-docs/).
+
+With sonobe, developers can choose which folding scheme and decider to use. The following code snippet shows how to fold a circuit using the Nova folding scheme and an EVM-compatible decider:
+
+```rust
```

Comment:
I'm unsure on having a partial example in this readme since we moved the code samples into [sonobe-docs](https://privacy-scaling-explorations.github.io/sonobe-docs/usage/overview.html).